extensions/youtube/src/components/popular_video.tsx

```diff
@@ -1,23 +1,38 @@
-import { showToast, Toast } from "@raycast/api";
+import { Action, Icon, Color, getPreferenceValues, showToast, Toast } from "@raycast/api";
 import { getErrorMessage } from "../lib/utils";
 import { getPopularVideos, useRefresher, Video } from "../lib/youtubeapi";
 import { VideoItem } from "./video";
-import { ListOrGrid, getViewLayout, getGridItemSize } from "./listgrid";
-import { PinVideo } from "./recent_videos";
+import { ListOrGrid } from "./listgrid";
+import { Preferences } from "../lib/types";
+import { addPinnedVideo } from "./recent_videos";
+
+const { griditemsize } = getPreferenceValues<Preferences>();
```

Comment:
What is the reason to call it outside of the function body?

Reply:
No reason. I moved them inside the function calls.